docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -72,7 +70,8 @@ Quadlet requires the use of cgroup v2, use `podman info --format {{.Host.Cgroups
 ### Service Type
 
 By default, the `Type` field of the `Service` section of the Quadlet file does not need to be set.
-Quadlet will set it to `notify` for `.container` and `.kube` files and to `oneshot` for `.volume`, `.network` and `.image` files.
+Quadlet will set it to `notify` for `.container` and `.kube` files,
+`forking` for `.pod` and `oneshot` for `.volume`, `.network` and `.image` files.
```

Comment:
This takes a few retries for my brain to parse. I think 'files' and a comma would help:
>... for `.pod` **files,** and ...

Reply:
Done